### PR TITLE
Visitor nav

### PR DESCRIPTION
**Pull request by WHomer**
What does this PR do? 
Resolves user story #81, notice the comments as the cart items number is pulled to a new issue/card.
<b>Notice</b>, possible temp routes and views for testing purpose.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -4,19 +4,41 @@
...
-    <ul class="main-nav">
-
-
-      <section class="register-link">
-        <%= link_to 'Register', register_path %>
-      </section>
-    </ul>
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <a class="navbar-brand" href="/">
+        <img class="cheese-icon" src="https://images.vexels.com/media/users/3/160051/isolated/lists/f9ce5cb5a15cc2cd4d5df19c4aff2858-cheese-piece-flat.png" width="30" height="30" alt="">
```
Comment (VinceCarollo) [inline app/views/layouts/application.html.erb:15]: I like the cheese button!

**Review by VinceCarollo — APPROVED** [review]
Solid Navbar Start!

File: config/routes.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
   resources :items, only: [:index, :show]
 
-  resources :carts, only: [:create]
+  resources :carts, only: [:create, :index] #added index as we need to show the cart -- User Story 2, Visitor Navigation
```
Comment (BrennanAyers) [inline config/routes.rb:7]: Cool, this makes sense.

File: config/routes.rb [hunk 3]
```diff
@@ -18,6 +18,8 @@
 
   get '/dashboard', to: 'merchants#show'
 
+  get '/merchants', to: 'merchants#index' # a link to see all merchants ("/merchants") -- User Story 2, Visitor Navigation
```
Comment (BrennanAyers) [inline config/routes.rb:21]: 👍

File: spec/features/navigation/application_spec.rb [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        visit root_path
+
+        within '.navbar' do
+          click_link('Register')
+        end
+        expect(current_path).to eq(register_path)
+      end
+    end
+  end
+end
```
Comment (BrennanAyers) [inline spec/features/navigation/application_spec.rb:61]: Spec looks like a good start!

**Review by zachlp2016 — APPROVED** [review]
Looks gouda.

**Pull request merged**